dockerFiles/marklogic-server-centos:base

```diff
@@ -92,6 +103,5 @@ VOLUME /var/opt/MarkLogic
 ###############################################################
 # set entrypoint
 ###############################################################
-
 ENTRYPOINT ["/usr/local/bin/start-marklogic.sh"]
 CMD ["/bin/bash"]
```

Comment:
I know this is out of the scope of the task to reduce the image size but can you check that this is needed? Won't this be passed as an argument to start-marklogic.sh which would be ignored?

Reply:
I read this article from [AWS](https://aws.amazon.com/blogs/opensource/demystifying-entrypoint-cmd-docker/) which helped explain the purpose and use case of both ENTRYPOINT and CMD. 

Basically we could use either and the choice between each (or using both) doesn't have a deep technical impact and is largely preferential. For the sake of simplicity of the Dockerfile I've removed the line since its superfluous.  

Updated: fde379c